agent/src/agent.php

```diff
@@ -108,6 +108,10 @@
     browser: $ingestDetailsBrowser,
     onAuthenticationSuccess: static fn (IngestDetails $ingestDetails, float $duration) => $info('Authentication successful ['.round($duration, 3).'s]'),
     onAuthenticationError: static fn (Throwable $e, float $duration) => $info('Authentication failed ['.round($duration, 3).'s]: '.$e->getMessage()),
+    onUnderQuota: static function () use (&$ingest) {
```

Comment:
Can we reference `$ingest` before it's declared?

Reply:
We sure can! It creates a variable reference that we then set below.